coins/src/adapters/tokenMapping.json

```diff
@@ -2441,6 +2441,30 @@
       "decimals": "18",
       "symbol": "aMerSOLVBTC",
       "to": "coingecko#solv-btc"
+    },
+    "0x7dcb50b2180bc896da1200d2726a88af5d2cbb5a": {
+      "decimals": "18",
+      "symbol": "ordi",
+      "to": "coingecko#ordi"
+    },
+    "0x7126bd63713a7212792b08fa2c39d39190a4cf5b": {
+      "decimals": "18",
+      "symbol": "esMP",
+      "to": "coingecko#merlinswap"
+    },
+    "0x4920fb03f3ea1c189dd216751f8d073dd680a136" : {
+      "decimals": "18",
+      "symbol": "SolvBTCSLP",
```

Comment:
are theses SLPs Uni V2 fork LPs? Can you explain how this is pegged to solvBTC?

Reply:
This _SLP (Surf LP token) token_ is not Uni V2 LP forks. They are _Delta Neutral Vaults_ LP token from Surf Protocol. The price of the LP peg with BTC. 

For example, a BTC pool with  `102` BTC in the pool (LP Deposits + Fees) with  `100` LP token issued will have a price of `1.02`

Details: https://surf-exchange.gitbook.io/docs/surf-protocol/liquidity-pools#lp-token

Reply:
Price Oracle: 

- SolvBTCSLP: https://scan.merlinchain.io/address/0x6717DC0D87a9BD6849F96948c29e8c8875c10096 (PythAggregatorV3)
- WBTCSLP: https://scan.merlinchain.io/address/0x6717DC0D87a9BD6849F96948c29e8c8875c10096 (PythAggregatorV3)
- MBTCSLP: https://scan.merlinchain.io/address/0x6717DC0D87a9BD6849F96948c29e8c8875c10096 (PythAggregatorV3)